database/migrations/2021_02_12_133858_create_recordings_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRecordingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('recordings', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('artist');
```

Comment:
Add foreign key, instead of using plain string
https://laravel.com/docs/5.0/schema#foreign-keys